insights/parsers/db2licm.py

```diff
@@ -0,0 +1,100 @@
+"""
+IBM DB2 Sever details
+=====================
+
+Module for the processing of output from the ``db2licm -l`` command.
+Data is avaliable as rows of the output contained
+in one ``Record`` object for each line of output.
```

Comment:
Seems there is no such `Record` in the following code. Would you please to update it correspondingly?